tests/conftest.py

```diff
@@ -31,13 +31,15 @@
     "*agent/test_google*",
 ]
 
-# ppc64le has limitations with some supported libraries.
+# ppc64le and s390x have limitations with some supported libraries.
 machine, py_version = get_runtime_env_info()
-if machine == "ppc64le":
-    collect_ignore_glob.append("*test_grpcio*")
+if machine in ["ppc64le", "s390x"]:
     collect_ignore_glob.append("*test_google-cloud*")
     collect_ignore_glob.append("*test_pymongo*")
```

Comment:
For ignoring multiple tests can we do something like below?

```suggestion
if machine in ["ppc64le", "s390x"]:
    collect_ignore_glob.extend(["*test_google-cloud*", "*test_pymongo*"])
```